Rohit/PostgresAssignment/src/services/class.ts

```diff
@@ -0,0 +1,20 @@
+const classSql = require('../database/dbconnect')
+
+exports.get_classes = async () => {
+  try {
+    await classSql.query("set search_path to myschool");
```

Comment:
no need to set search_path . Please remove it. Connect with Jatin/Tarun

Reply:
resolved